lametro/templates/partials/event_header_no_live_public_comment.html

```diff
@@ -0,0 +1,54 @@
+<div class="row">
```

Comment:
This is almost identical to the public comment partial, but omits ecomment and call in instructions. Defining these separately makes the conditional logic easier to follow.